tests/codebuild/run_integtest.sh

```diff
@@ -1,34 +1,25 @@
 #!/bin/bash
 
+source tests/codebuild/common.sh
+
 # TODOs
 # 1. Add validation for each steps and abort the test if steps fails
 # Build environment `Docker image` has all prerequisite setup and credentials are being passed using AWS system manager
 
 # Verbose trace of commands, helpful since test iteration takes a long time.
 set -x 
 
-function delete_tests {
-    # Stop jobs so we can do PrivateLink test.
-    kubectl delete hyperparametertuningjob --all
-    kubectl delete trainingjob --all
-    kubectl delete batchtransformjob --all
-    kubectl delete hostingdeployment --all
-    kubectl delete model --all
-}
-
 # A function to delete cluster, if cluster was not launched this will fail, so test will fail ultimately too
 function cleanup {
     # We want to run every command in this function, even if some fail.
     set +e
 
-    echo "Controller manager logs:"
-    kubectl -n sagemaker-k8s-operator-system logs "$(kubectl get pods -n sagemaker-k8s-operator-system | grep sagemaker-k8s-operator-controller-manager | awk '{print $1}')" manager
-
-    # Describe, if the test fails the Additional field might have more helpful info.
-    echo "trainingjob description:"
-    kubectl describe trainingjob
+    if [ "${PRINT_DEBUG}" != "false" ]; then
```

Comment:
nit: If we use `PRINT_DEBUG` as name for this flag, then our canaries and integration tests will always run in "debug mode" which feels weird. Consider `CI_MODE`?

Reply:
This only checks to see if PRINT_DEBUG!="false" to print the logs. That means the default usage is that it prints these logs. I only put this statement in so that a developer has the option to disable it - but they have to explicitly set that one variable.

I think a move to "developer mode" or the converse "ci mode"/"verbose mode" would be preferred for all defaults but it's outside the scope of this already bloated PR lol